extension/src/views/defaultProject/DefaultProjectsTreeDataProvider.ts

```diff
@@ -51,4 +52,22 @@ export class DefaultProjectsTreeDataProvider implements vscode.TreeDataProvider<
         }
         return [];
     }
+
+    private async getChildrenForProjectTreeItem(element: ProjectTreeItem): Promise<vscode.TreeItem[]> {
```

Comment:
here just copy the implementation from `GradleTasksTreeDataProvider.ts` since the method with the same name in that file should show pinned tasks now